core/scripts/cli/voting/votePhaseTiming.js

```diff
@@ -0,0 +1,20 @@
+const { VotePhasesEnum } = require("../../../../common/Enums");
+const moment = require("moment");
+const SECONDS_IN_DAY = 86400;
+const MINUTES_IN_HOUR = 60;
+
+module.exports = (contractTime, roundPhase) => {
```

Comment:
maybe add a comment noting what type you're expecting for `contractType` and `roundPhase`? Are they both strings? Would this work if they were BNs?

Reply:
ah yeah both strings